pkg/controller/catalog/server/k8s_handler.go

```diff
@@ -109,14 +101,14 @@ func (s *k8sHandler) serviceBindingCallback(e k8swatch.Event) error {
 }
 
 func (s *k8sHandler) serviceBrokerCallback(e k8swatch.Event) error {
-	var sb scmodel.ServiceBroker
+	var sb servicecatalog.Broker
 	err := util.TPRObjectToSCObject(e.Object, &sb)
 	if err != nil {
 		glog.Errorf("Failed to decode the received object %#v", err)
 		return err
 	}
 
-	if e.Type == "ADDED" || e.Type == "MODIFIED" {
+	if e.Type == k8swatch.Added {
```

Comment:
Fixed in https://github.com/kubernetes-incubator/service-catalog/pull/176/commits/5965fd2cb9b2a54f2e238fef661cc6d3cc4b507c